go.mod

```diff
@@ -1,6 +1,6 @@
 module sigs.k8s.io/aws-load-balancer-controller
 
-go 1.18
+go 1.19
```

Comment:
lets stick with 1.18 for the patch release. We can upgrade to the new version at a slower pace.

Reply:
make sense